config/pre-cache/cm.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
I assume this is just an example and that the UO would generate this configmap based on the contents of the manged policies ?